db/seeds/cip_seed.rb

```diff
@@ -11,81 +11,81 @@
-  ["83af890c-6092-40a4-87d1-b9f9575f3a95", "11: Deepening assessment, feedback and questioning", "## Time commitment\n\n- 1 hour self-directed study materials\n- 3.5 hours mentor sessions\n- 2.5 hours training sessions\n\n## Why this, why now?\n\nAs an NQT last year, you completed Block 5, which introduced you to what good assessment feedback and questioning are. Now in Year 2, we will support you to build on your learning and initial successes by deepening your knowledge in this area and developing further practical strategies. You will go beyond looking at what makes effective assessment to focus on how you can link assessment to metacognitive strategies. This builds on the work you did in Block 3 in this area. You will also look at how to ensure feedback is of high quality: that it is specific, helpful and drives learning forward.\n\n## Your workload\n\nAssessment and feedback can be time consuming; it can be disproportionately so. You will have found ways to make it more manageable in the work you did last year. In this Block, you will be encouraged to work with your colleagues to find more efficient approaches which will support you to manage your workload further. We will also spend some time looking specifically at peer and self-assessment. These strategies are often presented as ways to increase efficiency – however, the time saved is just time wasted unless pupils are doing this really well!\n\n## Activities in this Block\n\n<!-- wp:acf/product-section-index {\n    \"id\": \"block_5f3120aa3616b\",\n    \"name\": \"acf/product-section-index\",\n    \"mode\": \"auto\"\n} /-->\n\n", "15eb3adb-e0b0-4578-8ca4-56864b25855d", "d7ac2099-81cb-4f95-9c32-ca68775c61d0", 4],
+  ["ac3dc47f-e7fd-41b5-8c37-f096e4be451e", "Behaviour", "In this module you'll cover:\r\n\r\n* how to establish an effective learning environment\r\n* how to effectively manage behaviour\r\n* the importance of holding and promoting high expectations for all pupils", nil, "a4dc302c-ab71-4d7b-a10a-3116a778e8d5", 4],
+  ["e559d137-950b-4f5d-9878-8e14f0ea40db", "Instruction", "In this module you'll cover: \r\n\r\n* the link between effective instruction and pupil learning\r\n* the elements of explicit instruction and the ‘I-we-you’ model as a useful way of approaching this\r\n* how instruction can support more effective pupil thinking", "ac3dc47f-e7fd-41b5-8c37-f096e4be451e", "a4dc302c-ab71-4d7b-a10a-3116a778e8d5", 4],
```

Comment:
Was changing line endings a deliberate choice?

Reply:
Ah, well spotted, no, probably not. Will bring them back (at some point) :)